### DISCUSSION TITLE
如何使用 Hexo 和 Github Actions 搭建个人博客 | Becomin' Charles

### DISCUSSION THREAD

**Discussion by giscus** [charlestang/charlestang.github.io#5] [General]
# 如何使用 Hexo 和 Github Actions 搭建个人博客 | Becomin' Charles

缘起5 月 13 日晚间，一位 IT 互联网行业的前辈——陈皓（网名：左耳朵耗子）去逝。2004 年，从 Live Space 自定义脚本开始，我玩起了个人博客空间，到 2008 年正式建立自己的独立博客，很快就注意到了 CoolShell.cn 的个人博客，正是左耳朵耗子的博客。 他的博客内容原创性好，完整性强，有独到见解，是整个中文博客圈，数一数二的高质量博客。从他的博客我学习了很多的技术知

https://charlestang.github.io/2023/how-to-use-hexo-and-github-actions-to-build-a-personal-blog/?hmsr=toutiao.io&utm_campaign=toutiao.io&utm_medium=toutiao.io&utm_source=toutiao.io

**Comment by kimroniny** [comment]
楼主说的很有道理，每次写文章的时候成本很高

**Reply by charlestang** [reply]
如果你天天写，还能勉强跟上信息上的结构。不过，与之相对的，系统的代码也要时常更新并且维护，这就更难了。

**Reply by charlestang** [reply]
我做了一个客户端 [HexoPress](https://github.com/charlestang/HexoPress) 来解决这个问题。
